.travis.yml

```diff
@@ -9,6 +9,12 @@ matrix:
   include:
     - python: 2.7
       env: TOXENV=py27
+    - python: 3.3
```

Comment:
FTR: we no longer care for Python 3.3 in other projects

Reply:
I just removed py33 from the test. Thank you @kmike for confirming this 😄 The reason why I added py33 was because I saw it in w3lib test